numba/np/arraymath.py

```diff
@@ -3448,6 +3448,31 @@ def impl(a, b):
     return impl
 
 
+@overload(np.intersect1d)
+def jit_np_intersect1d(a, b):
+    # Not implemented to support assume_unique or return_indices
+    # return sorted, unique values in both arrays
+    if not (type_can_asarray(a) or type_can_asarray(b)):
+        raise TypingError('intersect1d: both inputs must be array-like')
+
+    def np_intersects1d_impl(a, b):
+        a = np.asarray(a)
+        b = np.asarray(b)
+        intersect = []
```

Comment:
Please do check what NumPy returns. From what I can tell, the NumPy implementation returns a NumPy array, not a list. It might also be worth using the typing section to figure out what the type of the returned array should be?